tests/core/test_monitoring.py

```
@@ -0,0 +1,169 @@
+"""
+Unit tests for the monitoring system.
+"""
+import pytest
+import asyncio
+import time
```

Ruff (0.8.2)

`time` imported but unused

Remove unused import: `time`

(F401)